dask/array/core.py

```diff
@@ -2249,6 +2258,120 @@ def to_zarr(arr, url, component=None, storage_options=None,
                      return_stored=return_stored)
 
 
+def from_tiledb(uri, attribute=None, chunks=None,
+                storage_options=None, **kwargs):
+    """Load array from the TileDB storage database
+
+    See https://docs.tiledb.io for more information.
+
+    Parameters
+    ----------
+    uri: TileDB array or str
+        Location of the data
+    """
```

Comment:
perhaps would be nice to have an examples section

Reply:
added!